doc/src/records/rsys.md

```diff
@@ -24,7 +24,7 @@ ACRESISTANCE, Compressor-based cooling and electric ('strip') heating. Primary h
 ASHP,  Air-source heat pump (compressor-based heating and cooling). Primary (compressor) heating input energy is accumulated to end use HTG of meter rsElecMtr. Auxiliary and defrost heating input energy is accumulated to end use HPBU of meter rsElecMtr or meter rsFuelMtr (depending on rsTypeAuxH).
 ASHPKGROOM,  Packaged room air-source heat pump.
 ASHPHYDRONIC, Air-to-water heat pump with hydronic distribution. Compressor performance is approximated using the air-to-air model with adjusted efficiencies.
-VCHP2, Air-to-air heat pump with variable speed compressor.
+VCHP3, Air-to-air heat pump with variable speed compressor.  Use rsPerfMapHtg and rsPerfMapClg to specify heating and cooling PERFORMANCEMAPs.
```

Comment:
This name still needs to be revised. "ASHPPERFMAP"?

We need PERFMAP versions for anything that is HP or AC.